cardano-cli/src/Cardano/CLI/Json/Friendly.hs

```diff
@@ -188,21 +188,11 @@ friendlyExtraKeyWits = \case
   TxExtraKeyWitnessesNone -> Null
   TxExtraKeyWitnesses _supported paymentKeyHashes -> toJSON paymentKeyHashes
 
--- | Special case of validity range:
--- in Shelley, upper bound is TTL, and no lower bound
-pattern ShelleyTtl
-  :: Maybe SlotNo -> (TxValidityLowerBound era, TxValidityUpperBound era)
-pattern ShelleyTtl ttl <-
-  ( TxValidityNoLowerBound
-  , TxValidityUpperBound _ ttl
-  )
```

Comment:
We don't do this anymore.  Allegra called the upper bound field "ttl" and had slightly different representation.  It represented no limit as `maxBound`.  Future eras represented no limit as `Nothing` instead.  We put a compatibility shim to make Allegra seem like it has the same presentation  - ie no limit means `Nothing` so we no longer have to do anything special here.